en/flight_controller/airlink.md

```diff
@@ -0,0 +1,1003 @@
+   <td>Debug console (internal connector)</td>
+   </tr>
+   <tr>
+   <td>Serial 6</td>
+   <td>USART 2</td>
+   <td>Telemetry 3</td>
+   </tr>
+   <tr>
+   <td>Serial 7</td>
+   <td>UART 4</td>
+   <td>External UART</td>
+   </tr>
+   </tbody>
+   </table>
+
+
+
+## RC Input
+
+![RC Input](../../assets/flight_controller/airlink/airlink-rc-input.jpg)
+
+RC input is configured on the SBUS pin and is connected to IO MCU via an inverter internally. 
+For PPM receivers please use RC Connector PPM pin located on the left side of the unit. 
+
+## Outputs
+
+AIRLink has 16 PWM ouputs. Main outputs 1-8 and connected to IO MCU. AUX outputs 1-8 are connected to FMU. 
+
+   <table>
+   <tbody>
+   <tr>
+   <th>Output </th>
+   <th>Timer </th>
+   <th>Channel </th>
+   </tr>
+   <tr>
+   <td>AUX 1</td>
+   <td>Timer 1</td>
+   <td>Channel 4</td>
+   </tr>
+   <tr>
+   <td>AUX 2</td>
+   <td>Timer 1</td>
+   <td>Channel 3</td>
+   </tr>
+   <tr>
+   <td>AUX 3</td>
+   <td>Timer 1</td>
+   <td>Channel 2</td>
+   </tr>
+   <tr>
+   <td>AUX 4</td>
+   <td>Timer 1</td>
+   <td>Channel 1</td>
+   </tr>
+   <tr>
+   <td>AUX 5</td>
+   <td>Timer 4</td>
+   <td>Channel 2</td>
+   </tr>
+   <tr>
+   <td>AUX 6</td>
+   <td>Timer 4</td>
+   <td>Channel 3</td>
+   </tr>
+   <tr>
+   <td>AUX 7</td>
+   <td>Timer 12</td>
+   <td>Channel 1</td>
+   </tr>
+   <tr>
+   <td>AUX 8</td>
+   <td>Timer 12</td>
+   <td>Channel 2</td>
+   </tr>
+   </tbody>
+   </table>
+
+DShot capabiltiy can be used on the first four AUX pins.
+
+## Building firmware
+
+:::tip
+Most users will not need to build this firmware!
+It is pre-built and automatically installed by *QGroundControl* when appropriate hardware is connected.
+:::
+
+To [build PX4](../dev_setup/building_px4.md) for this target:
+```
+make sky-drones_smartap-airlink
+```
+
+## Peripherals
+
+Peripherals information coming soon. 
+
+
+## Reference design
+
+![Reference design](../../assets/flight_controller/airlink/airlink-reference-design.png)
+
+AIRLink CAD model is available [here](https://docs.sky-drones.com/airlink/cad-model)
+
+AIRLink Reference design can be provided by request. Get in touch at [Sky-Drones contact page](https://sky-drones.com/contact-us)
+
+
+## More Information
+
+For more information and instructions on setting up and using the AIRLink system see  [AIRLink Documentation](https://docs.sky-drones.com/airlink/)
+
+For technical help, support and customization please get in touch at [Sky-Drones contact page](https://sky-drones.com/contact-us)
+
+More information can be found at  [www.sky-drones.com](https://sky-drones.com)
+
+Frequently asked questions are answered in [FAQ](https://docs.sky-drones.com/airlink/faq)
+
+
+## Where to Buy
+
+Purchase from the original Sky-Drones Store (worldwide shipping with 1-2 days order processing time): 
+
+ - [Buy AIRLink 4G](https://sky-drones.com/sets/airlink-enterprise-set.html)
+ - [Buy AIRLink 5G](https://sky-drones.com/sets/airlink-5g-enterprise-set.html)
```

Comment:
- Should these indicate edition? E.g. Airlink 4G Enterprise?
- Should there be a purchase link for the "Core"?